geometries/vel_earth_los/vel_earth_los_algorithm.py

```diff
@@ -1,32 +1,39 @@
-import os
-from pathlib import Path
 import spiceypy as sp
 
-def vel_earth_los(utc, galaxy_targ):
-    """Earth LOS velocity wrt target due to Earth's orbit around Sun. Should be positive when Earth and
-    target are receding from each other, and negative otherwise.
+def vel_earth_los(utc, instr):
+    """Earth LOS velocity wrt, and against, boresight due to Earth's
+    orbit around Sun. Should be negative when Earth is moving toward
+    boresight i.e. target side of image plane, be positive when Earth is
+    receding from boresight side of image plane, and zero when Earth is
+    moving within image plane
 
     Args:
-        utc (str): date and time at which Earth LOS velocity will be found
-        galaxy_targ (str): body ID for galaxy contained in mkfile
+        utc (str): date and time at which Earth LOS velocity is found
+        instr (str): instrument name e.g. ASP_SLIT1 or ASP_SLIT_0
 
     Returns:
         tuple: EARTH LOS velocity
     """
+
+    # # # # # PART 1: INSTRUMENT FRAME AND BORESIGHT VECTOR # # # # #
+
+    ref, bsight = sp.getfov(sp.bodn2c(instr),99,99,99)[1:3]
```

Comment:
Again, use instrument frame for vectors, and get observer from instrument frame info